databrowser/src/layouts/menu/ContentMenu.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <button v-if="dialogOpen" class="md:hidden" @click="closeDialog">
+    <IconClose />
+  </button>
+  <button v-else class="inline-flex space-x-4" @click="dialogOpen = true">
+    <IconMenu />
+    <span class="hidden md:block">{{ $t('header.menu.title') }}</span>
+  </button>
+
+  <Dialog
+    :open="dialogOpen"
+    class="fixed top-0 z-20 p-0 w-screen"
+    @close="closeDialog"
+  >
+    <DialogOverlay class="hidden md:block fixed inset-0 bg-black opacity-30" />
+    <!-- Desktop Menu -->
+    <div
+      class="
+        hidden
+        md:block
+        overflow-x-auto
+        absolute
+        top-0
+        left-0
+        h-screen
+        bg-white
+      "
+    >
+      <button class="block py-4 px-4" @click="closeDialog">
+        <IconClose />
+      </button>
+      <div class="inline-flex flex-row h-full divide-x-2">
+        <ContentMenuSubList
+          :items="content"
+          :title="$t('header.menu.allDatasets')"
```

Comment:
I would not make this value hard coded. This could also come from the configuration object passed to this object. This means that the passed data is a `MenuItem` rather than an array of `MenuItem`.